src/main/resources/assets/waila/lang/de_DE.lang

```diff
@@ -11,15 +12,19 @@ screen.button.back=Zurück
 screen.button.hidden=Versteckt
 screen.button.visible=Sichtbar
 screen.button.maintained=Gewartet
-screen.button.toggled=An/Aus
+screen.button.toggled=Umschalten
+screen.hwyla.warning=§cWarnung:§r Du verwendest eine Abspaltung(fork) von Waila auch genannt Hwyla. Du kannst die (zum größtenteils fertige) Liste der Änderungen sehen, durch das Klicken auf der Schaltfläche oben links in diesem Fenster.
 
 choice.showhidewaila=Zeige/Verstecke Waila
 choice.showhideidmeta=Zeige ID:Metadaten
-choice.toggledmaintained=An/Aus/Gewartet
+choice.toggledmaintained=Umschalten/Gewartet
 choice.showliquids=Flüssigkeiten
 choice.shifttoggledblock=Blöcke schleichend umschalten
 choice.shifttoggledents=Lebewesen schleichend umschalten
-choice.hidefromlist=Auf Spielerliste verstecken
+choice.hidefromlist=Verstecken vor der Spielerliste
+choice.hidefromdebug=Verstecken vor dem Fehlersuche-Menü
```

Comment:
"Debug" is a word that probably shouldn't be translated.

Reply:
Ok